js/sound.js

```diff
@@ -0,0 +1,48 @@
+const backgroundAudio = ["audio/music/epic1.mp3", "audio/music/relaxing1.mp3"];
```

Comment:
Please create a structure:

```
const backgroundAudio = [
    { "filename" : "audio/...",
       "legalNotice": "Song title - <a href="link to page of the sound file">Bensound.com</a>"
     }, ...
];

```

Reply:
Can I do that in a new PR? This can be first merged. Like sorry for this, but it just makes my work convenient.

Reply:
(2) If you add a folder named "bensound" and a LICENSE file in it which includes a notice that these files are CC-BY licensed as written in the license page of bensound, this should be fine.

Reply:
Done! Making a new PR.